README.md

```diff
@@ -18,11 +21,16 @@ I'm not sure. I wanted something natural, and it felt right. Plus, it's relative
 
 ### Roadmap
 
-I'm first aiming to finish broad support for categorical data—I want it to cover most, if not all of the SciKit Learn algorithms—at a minimum the non-highly complex ones (AUC might be a bit later). Plus, I'll add in some other useful once such as gini impurity.
-
+I'm first aiming to finish broad support for categorical data. Code quality improvements, and greater unit test coverage would also be preferred. Before getting to 1.0, support for types other than slices is desirable.
 
 ### Change log
 
+#### 0.3.0
```

Comment:
Don't forget to create releases on this repository as well, for consistency. I believe there are Cargo plugins for doing the whole release pipeline in a single command.

Reply:
Yes, will do as soon as `good-errors` is merged in 👍